components/config-provider/index.tsx

```diff
@@ -38,6 +35,11 @@ export const configConsumerProps = [
   'autoInsertSpaceInButton',
   'locale',
   'pageHeader',
+  'form',
+  'virtual',
+  'input',
+  'dropdownMatchSelectWidth',
+  'space',
```

Comment:
`configConsumerProps` is legacy usage since hooks not ready when coding this. May not extend this list since it may be removed in future.

Reply:
useMemo's deps has to be a const size array, so we need a const array for all the ConsumerProps keys. If `configConsumerProps` will be removed, should I create a new const array for useMemo's deps?

Reply:
Got it. you can use `rc-util/lib/hooks/useMemo` which provides additional compare function of shallow equal.